helpers/articleHelper.js

```diff
@@ -57,6 +57,16 @@ const createNewTags = async (tags) => {
   return tagIds;
 };
 
+
+const calcReadingTime = (bodyText) => {
+  const matches = bodyText.match(/\S+/g);
+  const numberOfWords = matches ? matches.length : 0;
+  const averageWPM = 225;
```

Comment:
What is `WPM`? Write it in full if it has to please.

Reply:
WPM means Words Per Minute. I will update it in my next PR. Thanks for the feedback.